src/software/networking/threaded_unix_listener.py

```diff
@@ -1,4 +1,3 @@
-import base64
```

Comment:
Why are we decoding and encoding on the C++ end now? Does this change the type of our protocol buffers?

Reply:
We are encoding and decoding in the c++ end because it really should be where that happens. The reasons why we are encoding is to get around g3log. we send the encoded string, so we will have to decode it in python. This is slow. If we send the decoded string, then it would be a bit better. 

This doesn't necessarily change the type of the protobuf.

Reply:
Gotcha, good find👍